src/database/models.py

```diff
@@ -80,7 +80,9 @@ def __str__(self):
 
 class UserReview(models.Model):
     review_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    organization = models.ForeignKey(Organization, on_delete=models.CASCADE)
+    donation = models.ForeignKey(
+        Donation, on_delete=models.CASCADE, null=True, blank=True
```

Comment:
Updating user review model to reference donation rather than organization (so that a review is tied to a specific donation)